sycl/include/sycl/detail/spirv.hpp

```diff
@@ -537,6 +537,48 @@ using EnableIfVectorShuffle =
     std::enable_if_t<detail::is_vector_arithmetic<T>::value, T>;
 #endif // ifndef __NVPTX__
 
+// Bitcast shuffles can be implemented using a single SubgroupShuffle
+// intrinsic, but require type-punning via an appropriate integer type
+#ifndef __NVPTX__
+template <typename T>
+using EnableIfBitcastShuffle =
+    detail::enable_if_t<!detail::is_arithmetic<T>::value &&
```

Comment:
nit: `std::enable_if_t` here and below

Reply:
Same for `_v` version instead of `::value`.

Reply:
done